.eslintrc.js

```diff
@@ -26,10 +26,6 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 8,
     sourceType: "module",
-    ecmaFeatures: {
-      jsx: false,
-      experimentalObjectRestSpread: true,
```

Comment:
This was to fix the following deprecation warning in the terminal whilst running <kbd>$ npm run eslint</kbd> (or <kbd>$ npm run lint</kbd>):

> (node:93334) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The `parserOptions.ecmaFeatures.experimentalObjectRestSpread` option is deprecated. Use `parserOptions.ecmaVersion` instead. (found in ".eslintrc.js")